src/components/bookComponents/renderbook.js

```diff
@@ -8,10 +8,22 @@ const Book = (props) => {
   const { book } = props;
   const dispatch = useDispatch();
   return (
-    <li id={book.item_id}>
-      <p className="book-title">{book.title}</p>
-      <p className="book-author">{book.category}</p>
-      <button type="submit" className="remove-btn" onClick={() => dispatch(removeBook(book))}>REMOVE</button>
+    <li id={book.item_id} className="book-sec">
+      <div className="book-info">
+        <span className="book-cat">
+          {` ${book.category}`}
+        </span>
+        <span className="book-name">
+          {` ${book.title} `}
+        </span>
+      </div>
+      <div className="btn-sec">
+        <button type="button" className="btn-c"> Comments </button>
+        <div className="line-2" />
+        <button type="submit" className="remove-btn btn-r" onClick={() => dispatch(removeBook(book))}>REMOVE</button>
+        <div className="line-2" />
+        <button type="button" className="btn-e"> Edit </button>
+      </div>
```

Comment:
- [ ] Currently the progress bar and chapter info are missing from each book. Please add them to match the given design.

Reply:
thanks